<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dial_1Asm</class>
 <widget class="QDialog" name="dial_1Asm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="AsmFreqPied">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="Label_AsmdelayTrig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay Trigger</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="AsmchoiceLogic">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>102</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>T&amp;RIG AND</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="AsmdelayTrig">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AsmStart">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AsmConfig">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Stats</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Pll</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AsmstatusDaq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Daq</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AsmstatusPll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="AsmBoard">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="Asmstatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="AsmStop">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>440</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AsmModeDaq">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mode Thor 3 mezzanines</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza0-Mezza1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza2-Mezza0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor Mezza2-Mezza1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And 3 Mezza</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And Mezza 0-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode Thor And Mezz 1-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mode ASM Pied</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label_AsmfreqPied">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Freq Pied</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="AsmConsole">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>521</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AsmFileName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AsmCalib">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFreq">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Mezza0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>M0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Mezza1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>M1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Mezza2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>M2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MonitoringValue">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Monitoring&lt;/p&gt;&lt;p&gt;Refresh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="SaveFile">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRunNb">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Nb</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="RunNb">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="SaveTest">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Hv">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>HV </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NbSamples">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>20</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelRunNb_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nb Samples</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PhaseDrsAdc">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>103</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift Clock DRS - ADC</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cfgRegDrs">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
